src/style.css

```
@@ -1,3 +1,76 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
+header {
+  background-color: #fff;
+  justify-content: space-between; /* Розподіляє елементи по сторонах */
```

Remove all comments, it's redundant here
```suggestion
  justify-content: space-between;
```